model/src/main/kotlin/config/LicenseFindingCuration.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import com.fasterxml.jackson.databind.annotation.JsonSerialize
+import com.fasterxml.jackson.databind.util.StdConverter
+
+/**
+ * A curation for license finding(s) applying to all license findings for which all matchers represented by this classes
+ * properties match. When applied it assigns the [concludedLicense] to all matching findings.
+ */
+data class LicenseFindingCuration(
+    /**
+     * A glob to match the file path of a license finding.
+     */
+    val path: String,
+    /**
+     * A matcher for the start line of a license finding, matches if the start line matches any of [startLines] or if
+     * [startLines] is empty.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @JsonDeserialize(converter = CsvStringToIntListConverter::class)
+    @JsonSerialize(converter = IntListToCsvStringConverter::class)
+    val startLines: List<Int> = emptyList(),
+    /**
+     * A matcher for the line count of a license finding, matches if the line count equals [lineCount] or if
+     * [lineCount] is null.
+     */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    val lineCount: Int? = null,
+    /**
+     * A matcher for the detected license of a license finding, matches if the detected license equals
+     * [detectedLicense].
+     */
+    val detectedLicense: String,
+    /**
+     * The concluded license as SPDX expression or [com.here.ort.spdx.SpdxLicense.NONE] for no license,
+     * see https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60.
+     */
+    val concludedLicense: String,
```

Comment:
Why not use `SpdxExpression?` as type for this? This would make sure that the string is a valid SPDX expression. We do the same in `PackageCurationData`.

Reply:
note from our discussion: ```detected``` licenses are still handled as ```String``` not as ```SPDX-Expression```.  As the applied curations are handled as ```detected license``` what you are asking for would require to handle ```detected licenses``` as ```SPDX Expression``` which is however a bigger topic which makes sense to be handled separatly.